javascript/javascript3/week1/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Weather app</title>
+    <script src="week1.js"></script>
```

Comment:
you have to move this script tag :`<script src="week1.js"></script>` in the buttom of your body .